app/src/main/java/com/github/steroidteam/todolist/view/adapter/ParentTaskAdapter.java

```diff
@@ -41,6 +42,7 @@ public ParentTaskAdapter(TodoAdapter.TaskCustomListener listener) {
         dateCategoryList.add("Next 5 days after Tomorrow");
         dateCategoryList.add("Later");
         dateCategoryList.add("Unknown");
+        dateCategoryList.add("Done");
```

Comment:
I think it could be a good thing to move these strings to strings.xml

Reply:
Sure you right!